composer.json

```diff
@@ -14,16 +14,17 @@
         }
     },
     "require": {
-        "php": "^5.5 || ^7.0",
-        "zendframework/zend-loader": "^2.5",
-        "zendframework/zend-stdlib": "^2.5 || ^3.0",
-        "zendframework/zend-uri": "^2.5",
-        "zendframework/zend-validator": "^2.5"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-loader": "^2.5.1",
```

Comment:
Why you think it should be outside the scope of this PR? IMHO it should be in this scope, because I've also added testing with lowest/locked/latest dependencies. Usually we bump to the latest version (which contain all updates - and previous version was not LTS) to force users to use versions with all fixes. I think we should then have these dependencies bump here. Otherwise we can find many things to split this PR into multiple smaller PRs, but what is the point?